### PR TITLE
Update eigensdk to 0.1.7

### PR DESCRIPTION
**Pull request by Hyodar**
This PR simply updates eigensdk to v0.1.7 and keeps compatibility.

### CODE REVIEW

File: aggregator/aggregator.go [hunk 1]
```diff
@@ -325,7 +326,7 @@ func (agg *Aggregator) Close() error {
 func (agg *Aggregator) sendAggregatedResponseToContract(blsAggServiceResp blsagg.BlsAggregationServiceResponse) {
 	if blsAggServiceResp.Err != nil {
 		agg.aggregatorListener.IncErroredSubmissions()
-		if errors.Is(blsAggServiceResp.Err, blsagg.TaskExpiredError) {
+		if strings.Contains(blsAggServiceResp.Err.Error(), "expired") {
```
Comment (emlautarom1) [inline aggregator/aggregator.go:329]: Is there no support for `errors.Is` for this particular one?

**Review by emlautarom1 — APPROVED** [review]
LGTM

File: aggregator/aggregator.go (hunk 1 above)
Comment (Hyodar) [reply aggregator/aggregator.go:329]: It's now an error function, so it's never the same error. This is honestly such a bummer, error handling should be easier in go. Not sure if there's a better way to do that, but seems unlikely

**Comment by Hyodar** [comment]
Proceeding with this to make it easier for new PRs to use new eigensdk features / fixes.

**Comment by taco-paco** [comment]
G move

**Pull request merged**